src/codeEditor/index.tsx

```diff
@@ -19,7 +36,7 @@ const CodeEditor = ({ className, code, lang, options, children, theme, ...attr }
 
   useEffect(() => {
     setMode(prevMode => {
-      const newMode = getMode(lang);
+      const newMode = getMode(lang as string);
```

Comment:
Should `lang` be required or should `getMode` be updated to support an undefined language? This feels like a code smell.